cli/bblfsh-sdk-tools/cmd/manifest.go

```diff
@@ -1,44 +1,27 @@
 package cmd
 
 import (
-	"fmt"
 	"path/filepath"
-	"strings"
 
 	"gopkg.in/bblfsh/sdk.v1/cli"
 	"gopkg.in/bblfsh/sdk.v1/manifest"
 )
 
-const ManifestCommandDescription = "" +
-	"prints the manifest as a list of variables ready to be evaluated by bash or make"
-
-type ManifestCommand struct {
+type manifestCommand struct {
```

Comment:
Manifest command is not a command anymore, just some helper code to read a manifest file, shouldn't it be named accordingly?

Reply:
This pattern is already present at https://github.com/bblfsh/sdk/blob/master/cli/common.go. I can change both if you want, for something like `Helper`

Reply:
As long as we're consistent, I'm fine with it. Whatever @juanjux prefers, I guess.